pipeline/coordinator.go

```diff
@@ -240,12 +241,12 @@ func (c *Coordinator) StartUploadJob(p UploadJobPayload) {
 		p.SourceFile = newSourceURL.String()
 		p.SignedSourceURL = signedURL
 		p.InputFileInfo = inputVideoProbe
-		p.GenerateMP4 = func(automp4 bool, duration float64) bool {
-			if automp4 && duration <= maxMP4OutDuration.Seconds() {
+		p.GenerateMP4 = func(forcemp4, automp4 bool, duration float64) bool {
+			if forcemp4 || (automp4 && duration <= maxMP4OutDuration.Seconds()) {
 				return true
 			}
 			return false
-		}(p.AutoMP4, p.InputFileInfo.Duration)
+		}(p.ForceMP4, p.AutoMP4, p.InputFileInfo.Duration)
```

Comment:
Not a change from this PR but curious about the function indirection. Why not?:

```suggestion
		p.GenerateMP4 = p.ForceMP4 || (p.AutoMP4 && p.InputFileInfo.Duration <= maxMP4OutDuration.Seconds())
```

Reply:
I had a couple more things happening in this function but since removed - will clean this up in next PR.